platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -47,28 +52,46 @@
 
   private static final Path BIN_DIR = Paths.get("bin");
 
+  /*
+   Files included in an import operation when upgrading to a newer version.
+  */
+  private static final List<String> SYSTEM_PROPERTIES_TO_PRESERVE =
+      ImmutableList.of("solr.password", "migration.supported.versions");
+
+  private static final Path CUSTOM_SYSTEM_PROPERTIES_PATH =
+      Paths.get("etc", "custom.system.properties");
+
+  private static final List<Path> UPGRADEABLE_SYSTEM_PATHS =
+      ImmutableList.of( //
+          Paths.get("etc", "users.properties"), Paths.get("etc", "users.attributes"));
+
+  /*
+   Files (in addition to the above) that are included under normal export and import operations
+   within the same product version.
+  */
   private static final List<Path> REQUIRED_SYSTEM_PATHS =
       ImmutableList.of( //
           Paths.get("etc", "ws-security"),
           Paths.get("etc", "system.properties"),
-          Paths.get("etc", "custom.system.properties"),
+          CUSTOM_SYSTEM_PROPERTIES_PATH,
```

Comment:
❓Any reason we've only defined a constant for this one?

Reply:
I just did it for this file because I needed a constant for only this file in the versionUpgradeImport. I'll see if I can do this in a cleaner way. I don't overly like it the way it is 🤔